11/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor/label-property-configuration.md

```diff
@@ -25,6 +25,19 @@ As well as the [default AngularJS filters](https://docs.angularjs.org/api/ng/fil
 | [umbCmsTitleCase](https://apidocs.umbraco.com/v10/ui/#/api/umbraco.filters.filter:umbCmsTitleCase) | Converts a string to title case                         | String                      |                                                              |
 | [umbCmsJoinArray](https://apidocs.umbraco.com/v10/ui/#/api/umbraco.filters.filter:umbCmsJoinArray) | Joins an array into one string                          | Array (of string or object) | separator: string used to join values together, e.g. "`, `"<br />prop (optional): string key indicating which property to join when used on an array of objects |  
 
+### Example usage
+
+#### mediaItemResolver
```

Comment:
I'm afraid H4 (####) is not going to be visible on the GitBook platform.

How about combining the two headings into something like this:

> ### Example: `mediaItemResolver`

What do you think?

Reply:
yep no issue with that